openelevationservice/server/api/querybuilder.py

```diff
@@ -147,14 +153,80 @@ def polygon_coloring_elevation(geometry, dataset):
     
     result_geom = query_final.scalar()
 
+
     # Behaviour when all vertices are out of bounds
     if result_geom == None:
         raise InvalidUsage(404, 4002,
                            'The requested geometry is outside the bounds of {}'.format(dataset))
-
+    
     return result_geom, [min_height, max_height], avg_height
+##End-Original code for the polygon_coloring_elevation function
```

Comment:
It is not necessary to comment on where a function starts and where it ends.

Reply:
Done. Deleted.